backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/lower/FunctionInlining.kt

```diff
@@ -241,10 +242,10 @@ internal class FunctionInlining(val context: Context) : IrElementTransformerVoid
                     }.apply {
                         functionParameters.forEach {
                             val argument =
-                                    if (!unboundArgsSet.contains(it))
-                                        boundFunctionParametersMap[it]!!
-                                    else
+                                    if (unboundArgsSet.contains(it))
                                         valueParameters[unboundIndex++].second
+                                    else
+                                        boundFunctionParametersMap[it]!!
```

Comment:
This change doesn't alter the code behaviour, does it?

Reply:
True, it's just more readable IMO.

Reply:
ok